src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/OrganisationChecks/WebsiteAddressTests.cs

```diff
@@ -0,0 +1,139 @@
+using SFA.DAS.AssessorService.Api.Types.Models.Validation;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.Tests.Controllers.Gateway.OrganisationChecks
+{
+    [TestFixture]
+    public class WebsiteAddressTests
+    {
+        private RoatpGatewayOrganisationChecksController _controller;
+        private Mock<IRoatpApplicationApiClient> _applyApiClient;
+        private Mock<IHttpContextAccessor> _contextAccessor;
+        private Mock<IRoatpGatewayPageViewModelValidator> _gatewayValidator;
+        private Mock<IGatewayOrganisationChecksOrchestrator> _orchestrator;
+        private Mock<ILogger<RoatpGatewayOrganisationChecksController>> _logger;
+
+        private string username = "john smith";
+        private string givenName = "john";
+        private string surname = "smith";
+
+        [SetUp]
+        public void Setup()
+        {
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _contextAccessor = new Mock<IHttpContextAccessor>();
+            _gatewayValidator = new Mock<IRoatpGatewayPageViewModelValidator>();
+            _logger = new Mock<ILogger<RoatpGatewayOrganisationChecksController>>();
+            _orchestrator = new Mock<IGatewayOrganisationChecksOrchestrator>();
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+             {
+                new Claim(ClaimTypes.NameIdentifier, "1"),
+                new Claim("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn", username),
+                new Claim(ClaimTypes.GivenName, givenName),
+                new Claim(ClaimTypes.Surname, surname)
+             }));
+
+            var context = new DefaultHttpContext { User = user };
+            _gatewayValidator.Setup(v => v.Validate(It.IsAny<LegalNamePageViewModel>()))
+                .ReturnsAsync(new ValidationResponse
+                {
+                    Errors = new List<ValidationErrorDetail>()
+                }
+                );
+            _contextAccessor.Setup(_ => _.HttpContext).Returns(context);
+            _controller = new RoatpGatewayOrganisationChecksController(_applyApiClient.Object, _contextAccessor.Object, _gatewayValidator.Object, _orchestrator.Object, _logger.Object);
+        }
+
+        [Test]
+        public void check_website_request_is_sent()
+        {
+            var applicationId = Guid.NewGuid();
+
+            _orchestrator.Setup(x => x.GetWebsiteViewModel(new GetWebsiteRequest(applicationId, username)))
+                .ReturnsAsync(new WebsiteViewModel())
+                .Verifiable("view model not returned");
+
+            var _result = _controller.GetWebsitePage(applicationId).Result;
+            _orchestrator.Verify(x => x.GetWebsiteViewModel(It.IsAny<GetWebsiteRequest>()), Times.Once());
```

Comment:
All .IsAnys need to be replaced with expected values.

Reply:
All .IsAnys are replaced with the expected values now